app-arch/brotli/brotli-1.0.2.recipe

```diff
@@ -0,0 +1,149 @@
+SUMMARY="A generic-purpose lossless compression algorithm"
+DESCRIPTION="A generic-purpose lossless compression algorithm that \
+compresses data using a combination of a modern variant of the LZ77 algorithm, \
+Huffman coding and 2nd order context modeling, with a compression ratio \
+comparable to the best currently available general-purpose compression methods. \
+It is similar in speed with deflate but offers more dense compression."
+HOMEPAGE="https://github.com/google/brotli"
+COPYRIGHT="2009, 2010, 2013-2016 Brotli Authors"
```

Comment:
s/2016/2017/

Also, it would be nice to rename the tarball by adding this line after CHECKSUM_SHA256:
```
SOURCE_FILENAME="brotli-$portVersion.tar.gz"
```

Reply:
It's [2016](https://github.com/google/brotli/blob/master/LICENSE#L1) :P

I don't see any compelling reason to rename the tarball, after all they're contained within their respective port folder

Reply:
As we can see in https://github.com/google/brotli/releases Brotli 1.0.2 was released on November 29, 2017. So it cannot be 2016 :)

Regarding the tarball, there are a few arguments in favor for the rename:
* Gentoo renames the tarball;
* We already do so in many similar cases;
* A tarball without the name of the program is not very meaningful.

That said, since the recipe is still correct without the rename, it is perfectly OK to keep the original filename if you wish to. That's one of the priviledges you have when you write a great recipe!